libs/ledgerjs/package.json

```diff
@@ -13,5 +12,15 @@
     "invariant": "^2.2.4",
     "js-levenshtein": "^1.1.6",
     "lodash": "^4.17.21"
+  },
+  "devDependencies": {
+    "@types/bs58check": "^2.1.0",
+    "@types/jest": "^29.5.4",
+    "@types/node": "^20.5.7",
+    "create-hash": "^1.1.3",
+    "eslint-plugin-react": "^7.32.2",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.1.1",
+    "ts-node": "^10.9.1"
```

Comment:
Re-opening, as you didn't answer the question :slightly_smiling_face: 

I'm seeing you could remove a few ones, but can't the remainder be in the `cryptoassets` package instead of ledgerjs?